### ISSUE TITLE
Syntax error in shell expression with missing tab indentation

### ISSUE THREAD

**Issue by epilys** [rmyorston/pdpmake#16]
Hello, I came across this case which gnu make and bsd make handle without error. I don't know if it's intended according to the POSIX spec, or a bug.

This is a minimal testcase. As you can see by the weird indentation, new lines within the `case` expressions start with space instead of tab.

```make
all:
	case "test" in \
  *es*) printf "yes";; \
  *) printf "no" ;; \
	esac
```

Whitespace characters annotated:

```make
all:⏎
⇥case "test" in \⏎
␣␣*es*) printf "yes";; \⏎
␣␣*) printf "no" ;; \⏎
⇥esac⏎
```

output:

```shell
$ make    
case "test" in \
sh: 1: Syntax error: end of file unexpected (expecting ")")
make: failed to build 'all'
```

**Comment by rmyorston** [comment]
Yes, its a bug. Try the latest code.

**Comment by rmyorston** [comment]
The fix for this is in the 1.4.0 release.
